workers/default/repository/tasks.py

```diff
@@ -56,6 +56,9 @@ def _initialize_object(obj, params, user):
         _add_files(obj, params['files'], user)
     if 'parts' in params:
         for part in params['parts']:
+            # TODO check if this is the proper solution
```

Comment:
Kann man machen, finde ich aber nicht so schön, weil die Lösung nicht generisch ist. Was wenn ein Template mal andere Metadaten des Root-Objekts in parts braucht? Dann muss man hier wieder den Code anpassen ...

Ich würde dafür plädieren, entweder die Metadaten der Parts im Frontend schon komplett zu setzen.